doc/source/whatsnew/v1.5.0.rst

```diff
@@ -227,7 +227,7 @@ MultiIndex
 I/O
 ^^^
 - Bug in :meth:`DataFrame.to_stata` where no error is raised if the :class:`DataFrame` contains ``-np.inf`` (:issue:`45350`)
--
+- Bug in :meth:`to_json` fix memory leak (:issue:`43877`)
```

Comment:
@vernetya can you move to 1.4 (if you can do tonight will merge) otherwise can put in 1.4.1